src/services/aws_macros.ts

```diff
@@ -82,6 +85,18 @@ export class AWS {
           ),
         ),
     });
+    // Get around some strange internal caching inside of codebuild of the IAM roles
```

Comment:
does it make sense to create a shared retry strategy for that, or needs to be a custom one?

Reply:
the `retryDecider` is specific to this strategy unfortunately